settings_dialog.js

```diff
@@ -1,41 +1,55 @@
 // Everything under the settings button
 "use strict";
 
+const fs = require('fs-extra');
 const path = require('path');
 
 var {shell} = require('electron');
+const {dialog} = require('electron').remote;
+const win = remote.getCurrentWindow();
+
+const {getVersionData} = require("./version_info.js");
 
 const { Modal, showGenericError} = require('./modal');
-const {listInstalledVersions, deleteInstalledVersion} = require('./install_handler.js');
-const { settings, saveSettings, installPath } = require('./settings.js');
+const { listInstalledVersions, deleteInstalledVersion } = require('./install_handler.js');
+
+const { settings, dataFolder, saveSettings, defaultInstallPath } = require('./settings.js');
```

Comment:
Again with `dataFolder` here. It is not being used, so why import it?

Reply:
Didn't notice that one before, that will be removed.